src/show/components/TextField/index.js

```diff
@@ -1,23 +1,83 @@
-import React from "react";
-
+import React, { useState } from 'react';
+import { ReactComponent as EyeIcon } from '../../assets/svg/eye-icon.svg';
```

Comment:
Don't do this here. Create a index.js file inside the svg and 
import { ReactComponent as EyeIcon } from '../../assets/svg/eye-icon.svg'

export const icons =[ EyeIcon ]

icons.map(icon => {
  const Icon = props => {
    const ShowIcon = icon;
    return <ShowIcon {...props} />;
  };
  return Icon;
});

export {
EyeIcon
}

Reply:
same for all the svg, we will use in the future